nyxl-theme/src/css/_portlet_decorator.scss

```diff
@@ -0,0 +1,173 @@
+.portlet-decorate .portlet-content {
+	background: #FFF;
+	border: none;
+}
+
+.portlet-barebone .portlet-content {
+	padding: 0;
+}
+
+.portlet-title-text {
+	color: $nyred-color;
+	font-family: $title-font;
+	font-size: x-large;
+
+	@include textStyle(0.5px, capitalize);
+}
+
+.portlet-content {
+	a {
+
+		.btn,
+		.btn-default,
+		.btn:hover,
+		.btn:focus {
+			color: #FFF !important;
+			font-family: $link-text;
+			font-weight: bold;
+			@include textStyle(2px, uppercase);	
+		}
+	}
+
+	.portlet-body {
+		font-family: $body-font;
+		a {
+			font-family: $body-font;
+			@include textStyle(0, none);
+		}
+	}
+
+	.rating-label .rating a {
+		font-family: "fontawesome-alloy";
+	}
+}
+
+.panel-default {
+	border-color: $brightblue-color;
+	border-radius: 8px 8px 0 0 !important;
+}
+
+.panel-default > .panel-heading {
+	border-color: $brightblue-color;
+}
+
+.panel-default > .panel-heading + .panel-collapse > .panel-body {
+	border: none;
+}
+
+.panel-default > .panel-footer + .panel-collapse > .panel-body {
+	border-bottom-color: $brightblue-color;
+}
+
+/* BLOG styles */
```

Comment:
CSS comments should be standardized.  We do not need to uppercase the entire word.  Just capitalize it should be enough.